app/archivesspace-public/app/lib/app_config.rb

```diff
@@ -44,6 +44,8 @@ def self.load_defaults
   end
 
   def self.reload
+    $CONFIG_DEFAULTS_LOADED = false
```

Comment:
This little change ensures the AppConfig is reloaded completely when Rails tries to hot-code replace any changes in .. so I didn't have to restart the application for model changes to apply!